### DISCUSSION TITLE
第6节课笔记(5班)

### DISCUSSION THREAD

**Discussion by GitEasonXu** [InternLM/Tutorial#447] [笔记]
第 6 节 | OpenCompass 大模型评测笔记

**Comment by GitEasonXu** [comment]
笔记：https://github.com/GitEasonXu/InternLM/blob/main/homework/course_6/coursework.md

**Comment by angellyao** [comment]
https://zhuanlan.zhihu.com/p/678917426

**Comment by ZhaoHongSen0726** [comment]
https://blog.csdn.net/m0_75085274/article/details/135727946?spm=1001.2014.3001.5502

**Comment by JayceNing** [comment]
https://zhuanlan.zhihu.com/p/679061970

**Comment by shengmouget** [comment]
[https://blog.csdn.net/shengweiit/article/details/135737894](https://blog.csdn.net/shengweiit/article/details/135737894)

**Comment by Farewell-CK** [comment]
https://github.com/8baby8/Farewell_Learn_puyu/blob/main/%E7%AC%AC%E5%85%AD%E8%8A%82%E8%AF%BE/note.md

**Comment by ATang0729** [comment]
https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class6-note/class6_note.md

**Comment by Milkve** [comment]
[第六节课笔记](https://juejin.cn/post/7329753721017597967)
